org.freedesktop.Platform.VulkanLayer.VkCapture.yml

```diff
@@ -0,0 +1,50 @@
+id: org.freedesktop.Platform.VulkanLayer.VkCapture
```

Comment:
Isn't this ID a bit too generic? Maybe `OBSVkCapture` could be better?

Reply:
Good point - now that you mention, I think `OBSVkCapture` is a better namespace for this plugin / Vulkan layer / etc.

Reply:
Yeah, hadn't considered that. I have changed it to `org.freedesktop.Platform.VulkanLayer.OBSVkCapture` now